Source/Additions/UIKit/UIDevice+BFKit.m

```diff
@@ -55,72 +55,88 @@ + (NSString * _Nonnull)devicePlatformString {
-    if ([platform isEqualToString:@"iPad2,6"])      return @"iPad mini (GSM)";
-    if ([platform isEqualToString:@"iPad2,7"])      return @"iPad mini (CDMA)";
-    if ([platform isEqualToString:@"iPad4,4"])      return @"iPad mini 2 (WiFi)";
-    if ([platform isEqualToString:@"iPad4,5"])      return @"iPad mini 2 (Cellular)";
-    if ([platform isEqualToString:@"iPad4,6"])      return @"iPad mini 2 (China)";
-    if ([platform isEqualToString:@"iPad4,7"])      return @"iPad mini 3 (WiFi)";
-    if ([platform isEqualToString:@"iPad4,8"])      return @"iPad mini 3 (Cellular)";
-    if ([platform isEqualToString:@"iPad4,9"])      return @"iPad mini 3 (China)";
-    if ([platform isEqualToString:@"iPad5,1"])      return @"iPad mini 4 (WiFi)";
-    if ([platform isEqualToString:@"iPad5,2"])      return @"iPad mini 4 (Cellular)";
-    // iPad Pro 9.7
-    if ([platform isEqualToString:@"iPad6,3"])      return @"iPad Pro 9.7 (WiFi)";
-    if ([platform isEqualToString:@"iPad6,4"])      return @"iPad Pro 9.7 (Cellular)";
-    // iPad Pro 12.9
-    if ([platform isEqualToString:@"iPad6,7"])      return @"iPad Pro 12.9 (WiFi)";
-    if ([platform isEqualToString:@"iPad6,8"])      return @"iPad Pro 12.9 (Cellular)";
+    if ([platform isEqualToString:@"iPad4,4"])      return @"iPad Mini 2 (WiFi)";
+    if ([platform isEqualToString:@"iPad4,5"])      return @"iPad Mini 2 (Cellular)";
+    if ([platform isEqualToString:@"iPad4,6"])      return @"iPad Mini 2";
+    if ([platform isEqualToString:@"iPad4,7"])      return @"iPad Mini 3";
+    if ([platform isEqualToString:@"iPad4,8"])      return @"iPad Mini 3";
+    if ([platform isEqualToString:@"iPad4,9"])      return @"iPad Mini 3";
+    if ([platform isEqualToString:@"iPad5,1"])      return @"iPad Mini 4 (WiFi)";
+    if ([platform isEqualToString:@"iPad5,2"])      return @"iPad Mini 4 (LTE)";
+    if ([platform isEqualToString:@"iPad5,3"])      return @"iPad Air 2";
+    if ([platform isEqualToString:@"iPad5,4"])      return @"iPad Air 2";
+    if ([platform isEqualToString:@"iPad6,3"])      return @"iPad Pro 9.7";
+    if ([platform isEqualToString:@"iPad6,4"])      return @"iPad Pro 9.7";
+    if ([platform isEqualToString:@"iPad6,7"])      return @"iPad Pro 12.9";
+    if ([platform isEqualToString:@"iPad6,8"])      return @"iPad Pro 12.9";
+    
+    if ([platform isEqualToString:@"iPad6,11"])      return @"iPad (5th Gen)";
+    if ([platform isEqualToString:@"iPad6,12"])      return @"iPad (5th Gen)";
+    if ([platform isEqualToString:@"iPad7,1"])      return @"iPad Pro 12.9 (2nd Gen)";
+    if ([platform isEqualToString:@"iPad7,2"])      return @"iPad Pro 12.9 (2nd Gen)";
+    if ([platform isEqualToString:@"iPad7,3"])      return @"iPad Pro 10.5";
+    if ([platform isEqualToString:@"iPad7,4"])      return @"iPad Pro 10.5";
+    
     // Apple TV
     if ([platform isEqualToString:@"AppleTV2,1"])   return @"Apple TV 2G";
```

Comment:
In according to [The iPhone Wiki](https://www.theiphonewiki.com/wiki/Models) is better to name it `Apple TV (2nd Gen)